src/Commands/AnalyseCommand.php

```diff
@@ -70,10 +78,36 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             $output
         );
 
-        $configurationFile = getcwd() . DIRECTORY_SEPARATOR . 'graphlint.php';
+        $currentWorkingDirectory = getcwd();
+        if ($currentWorkingDirectory === false) {
+            throw new ShouldNotHappenException();
+        }
+
+        /** @var string|null $configurationFile */
+        $configurationFile = $input->getOption('configuration');
+
+        if ($configurationFile === null) {
+            foreach (['graphlint.php'] as $discoverableConfigName) {
```

Comment:
Added a loop here so we can add additional default config names if needed in future (e.g. `.graphlint.php`)